src/test/java/ru/training/at/hw2/Ex1.java

```diff
@@ -0,0 +1,140 @@
+package ru.training.at.hw2;
+
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+
+public class Ex1 {
+
+    private WebDriverWait webDriverWait;
+    WebDriver webDriver;
+    WebElement webElement;
+    SoftAssert softAssert = new SoftAssert();
+
+    @Test
+    public void ex1Point1() {  //1.Open Test Site by Url
+        System.setProperty("webdriver.chrome.driver",
+                "/Users/Nikita/Downloads/chromedriver_win32/chromedriver.exe");
+        webDriver = new ChromeDriver();
+        webDriver.manage().window().maximize();
+        webDriver.manage().timeouts()
+                .implicitlyWait(10, TimeUnit.SECONDS);
+        webDriver.get("https://jdi-testing.github.io/jdi-light/index.html");
+        softAssert.assertEquals("https://jdi-testing.github.io/jdi-light/index.html",
+                webDriver.getCurrentUrl());
+    }
+
+    @Test
+    public void ex1Point2() {  //2.Assert Browser title
+        webElement = webDriver.findElement(By.xpath("//header"));
+        softAssert.assertEquals("Home Page",
+                webDriver.getTitle());
+    }
+
+    @Test
+    public void ex1Point3() {  //3. Perform login
+        webElement = webDriver.findElement(
+                By.cssSelector("ul.uui-navigation.navbar-nav.navbar-right"));
+        webElement.click();
+        webElement = webDriver.findElement(By.id("name"));
+        webElement.sendKeys("Roman");
+        webElement = webDriver.findElement(By.id("password"));
+        webElement.sendKeys("Jdi1234");
+        webElement = webDriver.findElement(By.cssSelector(
+                "button.uui-button.dark-blue.btn-login"));
+        softAssert.assertEquals("LOGOUT", webElement.getText());
+    }
+
+    @Test
+    public void ex1Point4() {  //4. Assert Username is loggined
+        webElement = webDriver.findElement(By.id("login-button"));
+        webElement.click();
+        webElement = webDriver.findElement(By.id("user-name"));
+        softAssert.assertEquals("ROMAN IOVLEV", webElement.getText());
+    }
+
+    @Test
+    public void ex1Point5() {  //5.Assert that there are 4...
+        List<WebElement> webElements = webDriver
+                .findElements(By.cssSelector("ul.uui-navigation.nav.navbar-nav.m-l8"));
+        softAssert.assertEquals("HOME\n"
+                + "CONTACT FORM\n"
+                + "SERVICE\n"
+                + "METALS & COLORS", webElements.get(0).getText());
```

Comment:
Consider the use of another selector, so that assertion compares a list of Strings instead of one long string

something like
```suggestion
        List<WebElement> webElements = webDriver.findElements(By.cssSelector("some other selector"));
        List<String> actualCollection = webElements.stream().map(WebElement::getText).collect(Collectors.toList());
        List<String> expectedCollection = Arrays.asList("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS");
        softAssert.assertEquals(webElements.get(0).getText(), expectedCollection);
```